### ISSUE TITLE
Default Accessibility should be NotApplicable (zero)

### ISSUE THREAD

**Issue by ChaseFlorell** [ChaseFlorell/CodeToNeo4j#10]
Right now the default accessibility is `Private`, which is good for the most part, but it doesn't cover the scenario whereby a codebase creates a member without an access modifier.

missing access modifiers should be handled by default without the special logic currently found in the OptionsBinder
